### PR TITLE
#2135: Prepare rest-api mapping to share outside of project

### LINKED ISSUE #2135: Prepare rest-api mapping to share outside of project
## Background
Developers from other teams needs examples of rest-api calls when developing against Enceladus rest-api.

## Feature
Full set of example calls for existing endpoints of Enceladus rest-api.

### PR DESCRIPTION
**Pull request by miroslavpojer**
#2135: Prepare rest-api mapping to share outside of project
* implemented rest-api calls in Postman
* supported rest-api v2 and v3 including admin endpoints

Closes #2135

### CODE REVIEW

File: examples/rest_api/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- `02-Enceladus.rest-api.v2.postman_collection.json` - define all rest-api v2 compatible calls
+- `03-Enceladus.rest-api.v3.postman_collection.json` - define all rest-api v3 compatible calls
+
+## Setup
+
+### Install Postman
+Download and install Postman from https://www.postman.com/downloads.
+
+### Import into Postman
+Import variables first. Rest of files depends on your needs.
```
Comment (benedeki) [inline examples/rest_api/README.md:31]: I had little issue to find the proper import method (Import button too big 🤣 ), definitely harder then downloading postman, which has its link mentioned. Maybe a little hint to use the button on the main page.... 👼

**Review by benedeki — COMMENTED** [review]
- [x] code reviewed
- [x] pulled
- [x] imported
- [ ] run

File: examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
+{
```
[the remark this answers is not in the document]
Comment (miroslavpojer) [reply examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json:1]: That import file holds all used variables in example endpoint calls. The name is related to my used environment - IntelliJ's Tomcat. Anybody can duplicate this set env. setting after import and rename and change variable values to his customs.

File: examples/rest_api/README.md (hunk 1 above)
Comment (miroslavpojer) [inline examples/rest_api/README.md:31]:
```suggestion
Import variables first. Rest of files depends on your needs. 
- For import use `Import` button visible next to 'My Workspace' in the left panel.
- Warning: If left panel is minimized the button is not visible.
```

File: examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+			"key": "newCheckpointName",
+			"value": "AnotherSource",
+			"type": "default",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2022-10-05T18:42:46.410Z",
+	"_postman_exported_using": "Postman/9.30.4"
+}
```
Comment (benedeki) [inline examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json:214]: EOF

Comment (miroslavpojer) [inline examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json:214]:
```suggestion
}

```

File: examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (benedeki) [reply examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json:1]: Yes, but _IntelliJ_ has no relation to the variables, nor is used either in variable name nor in values. The only link I see, is that the API application was run from _IntelliJ_

So the filename is misleading. Might make the impression that one needs the _IntelliJ_ (and Tomcat) to use the file. 🤔 

I would really just name the file in some neutral name like `Enceladus.local.postman_environment.json`

[the remark this answers is not in the document]
Comment (miroslavpojer) [reply examples/rest_api/01-Local_IntellijJ Tomcat.postman_environment.json]: Solved in 0b545659bb34cba5558447ed37ebd13eb4973727

**Review by benedeki — APPROVED** [review]

**Pull request merged**